services/head_tracker_test.go

```diff
@@ -44,7 +45,7 @@ func TestHeadTracker_Get(t *testing.T) {
 		wantError bool
 	}{
 		{"greater", start, cltest.IndexableBlockNumber(6), big.NewInt(6), false},
-		{"less than", start, cltest.IndexableBlockNumber(1), big.NewInt(5), false},
+		{"less than", start, cltest.IndexableBlockNumber(1), big.NewInt(5), true},
```

Comment:
Was this a bug?

Reply:
I wouldn't go that far, but I would say that I've changed the head tracker behavior to return an error when we try to save a block less than the head. It just ends up being `logger.Error(...)` out right now.